measures/ResStockArguments/measure.rb

```diff
@@ -27,12 +16,12 @@ def name
 
   # human readable description
   def description
-    return 'Measure that pre-processes the arguments passed to the BuildResidentialHPXML measure.'
+    return 'Measure that pre-processes the arguments passed to the BuildResidentialHPXML and BuildResidentialScheduleFile measures.'
   end
 
   # human readable description of modeling approach
   def modeler_description
-    return ''
+    return 'Passes in all arguments from the options lookup, processes them, and then registers values to the runner to be used by other measures.'
```

Comment:
I'm not seeing that it registers any values. Didn't we revert that approach?

Reply:
Here? https://github.com/NREL/resstock/blob/develop/measures/ResStockArguments/measure.rb#L774. What were you thinking we reverted to doing?

Reply:
Ah, I was searching for runner.registerValue.

I thought you added this as part of the first approach for [this PR](https://github.com/NREL/resstock/pull/848), but then we came up with a different approach so that it's no longer needed. And I thought we specifically wanted to revert it because it causes a whole bunch of extra output columns to be produced. But maybe I'm confusing it with something else.

Reply:
Yes, these are separate issues. Anything registered from ResStockArguments gets "thrown away" since we create a `new_runner`: https://github.com/NREL/resstock/blob/develop/measures/BuildExistingModel/measure.rb#L261-L282. (Before the additional properties solution for the PR above, I wanted to put stuff on the main `runner`.)